pages/common/curl.md

```diff
@@ -5,32 +5,40 @@
 
 - Download a URL to a file:
```

Comment:
Well, if this makes it clearer to understand, we can update the wget page too. :)